adobe/react-spectrum
# fix(#4108): Picker: isLoading state should include "Loading..." in the element description

Closes #4108 

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue 4108](https://github.com/adobe/react-spectrum/issue/4108).
- [x] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.2/)

## 📝 Test Instructions:

1. Open [Picker: isLoading, no items story](https://reactspectrum.blob.core.windows.net/reactspectrum/68095c1c1c527f87345c516db44f078dce3e8304/storybook/index.html?path=/story/picker--loading&providerSwitcher-express=false).
2. Open DevTools and inspect the Picker trigger button to verify that it has an `aria-describedby` attribute that includes the IDREF for the ProgressCircle.
3. Verify under the Accessibility properties panel for the Picker trigger button, that the value for Description is "Loading..."

## 🧢 Your Project:

Adobe/Accessibility
